helm/argocd-test/action.yml

```diff
@@ -33,8 +58,17 @@ runs:
           username: ${{ github.actor }}
           password: ${{ inputs.github-token }}
 
-      - name: Run helm template
+      - name: Run helm template then kubeconform
```

Comment:
Small nit: I think it would be nicer to run kubeconform in a separate step so that it became clearer if it would be the `helm template` command or kubeconform failing otherwise lgtm :+1:

Reply:
done and working https://github.com/LedgerHQ/vault-lam/actions/runs/7986576868/job/21808909637